syntax = "proto3";

import "alice/boltalka/generative/service/proto/common.proto";

package NGenerativeBoltalka.Proto;

message TPHeadRequest {
    reserved 2;
    string Text = 1;
    oneof Location {
        string PtuneYtPath = 3;
        string PtuneS3ObjectName = 4;
    }
}

message TPHeadResponse {
    repeated float Scores = 1;
}

